backend/src/v5/schemas/tickets/templates.js

```diff
@@ -153,9 +159,13 @@ const configSchema = Yup.object().shape({
 	defaultView: defaultFalse,
 	defaultImage: Yup.boolean().when('defaultView', (defaultView, schema) => (defaultView ? schema.strip() : defaultFalse)),
 	pin: Yup.lazy((val) => (val?.color ? Yup.object({ color: pinColSchema }) : defaultFalse)),
+	status: Yup.lazy((val) => (val ? Yup.object({
+		values: Yup.array().of(customStatus).min(1).required(),
+		default: Yup.mixed().when('values', (values) => Yup.string().oneOf(values.map((v) => v.name)).required()),
```

Comment:
We use spread operator when possible so we should do it here too for consistency
```suggestion
		default: Yup.mixed().when('values', (values) => Yup.string().oneOf(values.map(({name}) => name)).required()),
```